app/src/componentLibrary/breadcrumbs/breadcrumb/breadcrumb.jsx

```diff
@@ -18,33 +18,29 @@ import React, { useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import { NavLink } from 'components/main/navLink';
-import { withTooltip } from 'componentLibrary/tooltip';
 import styles from './breadcrumb.scss';
 
 const cx = classNames.bind(styles);
 
-const BreadcrumbTitleComponent = ({
-  descriptor: { link, title, onClick },
+export const Breadcrumb = ({
   maxBreadcrumbWidth,
   titleTailNumChars,
-  tooltipDisabled,
-  setTooltipDisabled,
+  descriptor: { title, link, onClick },
 }) => {
   const ref = useRef();
+  const [tooltipDisabled, setTooltipDisabled] = useState(true);
```

Comment:
May be change `tooltipDisabled` to `breadcrumbTitle` with initial value null and remove tooltipDisabled check in str43?